internal/graph/check_test.go

```diff
@@ -2998,3 +3000,329 @@ func TestIteratorReadStartingFromUser(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildTupleKeyConditionFilter(t *testing.T) {
+	tests := []struct {
+		name         string
+		tupleKey     *openfgav1.TupleKey
+		model        *openfgav1.AuthorizationModel
+		context      map[string]interface{}
+		conditionMet bool
+		expectedErr  error
+	}{
+		{
+			name:     "no_condition",
+			tupleKey: tuple.NewTupleKey("document:1", "can_view", "user:maria"),
+			model: parser.MustTransformDSLToProto(`
+				model
+					schema 1.1
+
+				type user
+
+				type document
+					relations
+						define can_view: [user]`),
+			context:      map[string]interface{}{},
+			conditionMet: true,
+			expectedErr:  nil,
+		},
+		{
+			name:     "condition_not_met",
+			tupleKey: tuple.NewTupleKeyWithCondition("document:1", "viewer", "user:maria", "correct_ip", nil),
+			model: parser.MustTransformDSLToProto(`
+				model
+					schema 1.1
+
+				type user
+
+				type document
+					relations
+						define can_view: [user]`),
+			context:      map[string]interface{}{},
+			conditionMet: false,
+			expectedErr:  condition.NewEvaluationError("correct_ip", fmt.Errorf("condition was not found")),
+		},
+		{
+			name:     "condition_missing_parameter",
```

Comment:
given this test name don't you need to define the condition in the model? otherwise this test is the same as the previous one

Reply:
remove redundant call